src/models.ts

```diff
@@ -78,13 +78,20 @@ export const validateLoad = validate(loadSchema, {
   }
 });
 
+export interface IReport {
+  id: string;
+  displayName: string;
+}
+
 export interface IPage {
   name: string;
   displayName: string;
 }
 
 export interface IVisual {
   name: string;
```

Comment:
Similar to my other answer about `value` key in the `IDataValueReference`.

I would prefer to keep it as required and explicitly assign it `null` or leave it off the interface so we can add it later since going from optional to required is breaking change.

Also, if we leave it, we can move forward with the experimentation on `getVisuals` and `visual.setFilters`